website/docs/r/firewall_policy.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_firewall_policy"
+description: |-
+  Manages a Firewall Policy.
+---
+
+# azurerm_firewall_policy
+
+Manages a Firewall Policy.
+
+## Example Usage
+
+```hcl
+resource "azurerm_firewall_policy" "example" {
+  name                = "example"
+  resource_group_name = "example"
+  location            = "West Europe"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The Azure Region where the Firewall Policy should exist. Changing this forces a new Firewall Policy to be created.
+
+* `name` - (Required) The name which should be used for this Firewall Policy. Changing this forces a new Firewall Policy to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Firewall Policy should exist. Changing this forces a new Firewall Policy to be created.
+
+---
+
+* `base_policy_id` - (Optional) The ID of the base Firewall Policy.
+
+* `threat_intelligence_mode` - (Optional) The operation mode for Threat Intelligence. Possible values are `Alert`, `Deny` and `Off`.
```

Comment:
Should we mention the `default` value here?

```suggestion
* `threat_intelligence_mode` - (Optional) The operation mode for Threat Intelligence. Possible values are `Alert`, `Deny` and `Off`. Defaults to `Alert`.
```

Reply:
Thank you for pointing out!